src/components/icon/icon.scss

```diff
@@ -25,7 +25,9 @@ $icon-xx-small: 12px;
 
   box-sizing: content-box !important;
 }
-
+.icon-inner {
+  height: 100%;
```

Comment:
Don't we have some class called `h-100` here?

Reply:
no, we're trying to use a clear and declarative css and by now we have all the styles only in the css files